pymc3/distributions/transforms.py

```diff
@@ -105,12 +105,27 @@ def jacobian_det(self, x):
 log = Log()
 
 
+class Softplus(ElemwiseTransform):
```

Comment:
Not sure Softplus is the right name, above we name things according to the forward transform. `LogExp`, `SoftMinus`?